bin/emqx

```diff
@@ -143,11 +143,7 @@ if [ "$ES" -ne 0 ]; then
     exit $ES
 fi
 
-if [ -z "$WITH_EPMD" ]; then
-    EPMD_ARG="-start_epmd false -epmd_module ekka_epmd -proto_dist ekka"
-else
-    EPMD_ARG="-start_epmd true $PROTO_DIST_ARG"
```

Comment:
Q: is this removal intentional?  I thought that only in 5.0 we dropped support for epmd.

Reply:
we never really supported it since 4.2

Reply:
restored the support. 
just to keep it compatible.